common/baselines/librispeech/ls100/gmm/baseline_args.py

```diff
@@ -0,0 +1,428 @@
+"""
+Definition of all changeable pipeline parameters of the GMM-RASR-Pipeline
+
+The parameters defined here are based on the past experience with training LibriSpeech-100h and 960h models,
+but no excessive tuning has been done.
+"""
+from i6_core.features.filterbank import filter_width_from_channels
+from i6_core import cart
+
+from i6_experiments.common.setups.rasr import gmm_system
+
+from i6_experiments.common.setups.rasr.util import *
+from i6_experiments.users.luescher.cart.librispeech import (
+    FoldedCartQuestions,
+    UnfoldedCartQuestions,
+)
+
+
+def get_init_args():
+    dc_detection = False
+    samples_options = {
+        "audio_format": "wav",
+        "dc_detection": dc_detection,
+    }
+
+    am_args = {
+        "state_tying": "monophone",
+        "states_per_phone": 3,
+        "state_repetitions": 1,
+        "across_word_model": True,
+        "early_recombination": False,
+        "tdp_scale": 1.0,
+        "tdp_transition": (3.0, 0.0, 30.0, 0.0),  # loop, forward, skip, exit
+        "tdp_silence": (0.0, 3.0, "infinity", 20.0),
+        "tying_type": "global",
+        "nonword_phones": "",
+        "tdp_nonword": (
+            0.0,
+            3.0,
+            "infinity",
+            6.0,
+        ),  # only used when tying_type = global-and-nonword
+    }
+
+    costa_args = {"eval_recordings": True, "eval_lm": True}
+
+    feature_extraction_args = {
+        "mfcc": {
+            "num_deriv": 2,
+            "num_features": None,  # confusing name: number of max features, above number -> clipped
+            "mfcc_options": {
+                "warping_function": "mel",
+                # to be compatible with our old magic number, we have to use 20 features
+                "filter_width": filter_width_from_channels(
+                    channels=20, warping_function="mel", f_max=8000
+                ),
+                "normalize": True,
+                "normalization_options": None,
+                "without_samples": False,
+                "samples_options": samples_options,
+                "cepstrum_options": {
+                    "normalize": False,
+                    "outputs": 16,  # this is the actual output feature dimension
+                    "add_epsilon": not dc_detection,  # when there is no dc-detection we can have log(0) otherwise
+                    "epsilon": 1e-10,
+                },
+                "fft_options": None,
+            },
+        },
+        "energy": {
+            "energy_options": {
+                "without_samples": False,
+                "samples_options": samples_options,
+                "fft_options": {},
+            }
+        },
+    }
+
+    return RasrInitArgs(
+        costa_args=costa_args,
+        am_args=am_args,
+        feature_extraction_args=feature_extraction_args,
+        default_mixture_scorer_args={"scale": 0.3},
```

Comment:
does this set the emission model scale to 0.3? Did we ever use a value different from 1.0?

Reply:
Was not used anyway, removed.